### ISSUE TITLE
rbx-3 dependency on Mint 19.1

### ISSUE THREAD

**Issue by bilge-Kagan** [rvm/rvm#4641]
It may be duplicate, but i couldn't find solution for this issue.

```
$ rvm install rbx-3
Searching for binary rubies, this might take some time.
No binary rubies available for: mint/19.1/x86_64/rbx-3.107.
Continuing with compilation. Please read 'rvm help mount' to get more information on binary rubies.
Checking requirements for mint.
Installing requirements for mint.
Updating system - please wait
Installing required packages: libssl-dev, libssl-dev, clang-3.6, llvm-3.6, llvm-3.6-dev - please wait
Error running 'requirements_debian_libs_install libssl-dev libssl-dev clang-3.6 llvm-3.6 llvm-3.6-dev',
please read /home/bilgekagan/.rvm/log/1552693554_rbx-3.107/package_install_libssl-dev_libssl-dev_clang-3.6_llvm-3.6_llvm-3.6-dev.log
Requirements installation failed with status: 100.
```

**/home/bilgekagan/.rvm/log/1552693554_rbx-3.107/package_install_libssl-dev_libssl-dev_clang-3.6_llvm-3.6_llvm-3.6-dev.log >>**

```
+requirements_debian_libs_install:2> __rvm_try_sudo apt-get --no-install-recommends --yes install libssl-dev libssl-dev clang-3.6 llvm-3.6 llvm-3.6-dev
+__rvm_try_sudo:2> typeset -a command_to_run
+__rvm_try_sudo:3> typeset sudo_path sbin_path missing_paths
+__rvm_try_sudo:4> command_to_run=( apt-get --no-install-recommends --yes install libssl-dev libssl-dev clang-3.6 llvm-3.6 llvm-3.6-dev ) 
+__rvm_try_sudo:5> ((  UID == 0  ))
+__rvm_try_sudo:6> case 4 (0)
+__rvm_try_sudo:6> case 4 (1)
+__rvm_try_sudo:6> case 4 (2)
+__rvm_try_sudo:6> case 4 (*)
+__rvm_try_sudo:21> is_a_function __rvm_sudo
+is_a_function:2> typeset -f __rvm_sudo
+__rvm_try_sudo:23> missing_paths='' 
+__rvm_try_sudo:24> sbin_path=/sbin
+__rvm_try_sudo:26> [[ -d /sbin ]]
+__rvm_try_sudo:26> [[ :/home/bilgekagan/.rvm/gems/ruby-2.3.1/bin:/home/bilgekagan/.rvm/gems/ruby-2.3.1@global/bin:/usr/share/rvm/rubies/ruby-2.3.1/bin:/home/bilgekagan/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/share/rvm/bin:/home/bilgekagan/Programs/cmus_2.8.0/bin/: != *:/sbin:* ]]
+__rvm_try_sudo:24> sbin_path=/usr/sbin
+__rvm_try_sudo:26> [[ -d /usr/sbin ]]
+__rvm_try_sudo:26> [[ :/home/bilgekagan/.rvm/gems/ruby-2.3.1/bin:/home/bilgekagan/.rvm/gems/ruby-2.3.1@global/bin:/usr/share/rvm/rubies/ruby-2.3.1/bin:/home/bilgekagan/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/share/rvm/bin:/home/bilgekagan/Programs/cmus_2.8.0/bin/: != *:/usr/sbin:* ]]
+__rvm_try_sudo:24> sbin_path=/usr/local/sbin
+__rvm_try_sudo:26> [[ -d /usr/local/sbin ]]
+__rvm_try_sudo:26> [[ :/home/bilgekagan/.rvm/gems/ruby-2.3.1/bin:/home/bilgekagan/.rvm/gems/ruby-2.3.1@global/bin:/usr/share/rvm/rubies/ruby-2.3.1/bin:/home/bilgekagan/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/share/rvm/bin:/home/bilgekagan/Programs/cmus_2.8.0/bin/: != *:/usr/local/sbin:* ]]
+__rvm_try_sudo:30> [[ -n '' ]]
+__rvm_try_sudo:33> command_to_run=( __rvm_sudo -p '%p password required for '\''apt-get --no-install-recommends --yes install libssl-dev libssl-dev clang-3.6 llvm-3.6 llvm-3.6-dev'\'': ' apt-get --no-install-recommends --yes install libssl-dev libssl-dev clang-3.6 llvm-3.6 llvm-3.6-dev ) 
+__rvm_try_sudo:40> __rvm_sudo -p '%p password required for '\''apt-get --no-install-recommends --yes install libssl-dev libssl-dev clang-3.6 llvm-3.6 llvm-3.6-dev'\'': ' apt-get --no-install-recommends --yes install libssl-dev libssl-dev clang-3.6 llvm-3.6 llvm-3.6-dev
+__rvm_sudo:2> sudo -p '%p password required for '\''apt-get --no-install-recommends --yes install libssl-dev libssl-dev clang-3.6 llvm-3.6 llvm-3.6-dev'\'': ' apt-get --no-install-recommends --yes install libssl-dev libssl-dev clang-3.6 llvm-3.6 llvm-3.6-dev
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package llvm-3.6
E: Couldn't find any package by glob 'llvm-3.6'
E: Couldn't find any package by regex 'llvm-3.6'
E: Unable to locate package llvm-3.6-dev
E: Couldn't find any package by glob 'llvm-3.6-dev'
E: Couldn't find any package by regex 'llvm-3.6-dev'
+__rvm_try_sudo:40> return 100
+requirements_debian_libs_install:2> return 100
```

I've all dependencies's newer version. However, it wants, for example `llvm-3.6`, but there is no `llvm` package as **3-6** versioned. There is same issue rest of all dependencies. What can i do?

```
$ rvm --version
rvm 1.29.7-next (master) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]
```

**Comment by pkuczynski** [comment]
Which version of Mint do your run and which version of llvm is available?

**Comment by bilge-Kagan** [comment]
@pkuczynski 
`Mint 19.1`
`Linux 4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019 x86_64 GNU/Linux`
```
i   llvm                                                - Low-Level Virtual Machine (LLVM)                              
p   llvm:i386                                           - Low-Level Virtual Machine (LLVM)                              
p   llvm-3.7-doc                                        - Modular compiler and toolchain technologies, documentation    
p   llvm-3.9                                            - Modular compiler and toolchain technologies                   
p   llvm-3.9:i386                                       - Modular compiler and toolchain technologies                   
p   llvm-3.9-dev                                        - Modular compiler and toolchain technologies, libraries and hea
p   llvm-3.9-dev:i386                                   - Modular compiler and toolchain technologies, libraries and hea
p   llvm-3.9-doc                                        - Modular compiler and toolchain technologies, documentation    
p   llvm-3.9-examples                                   - Modular compiler and toolchain technologies, examples         
p   llvm-3.9-runtime                                    - Modular compiler and toolchain technologies, IR interpreter   
p   llvm-3.9-runtime:i386                               - Modular compiler and toolchain technologies, IR interpreter   
p   llvm-3.9-tools                                      - Modular compiler and toolchain technologies, tools            
p   llvm-3.9-tools:i386                                 - Modular compiler and toolchain technologies, tools            
p   llvm-4.0                                            - Modular compiler and toolchain technologies                   
p   llvm-4.0:i386                                       - Modular compiler and toolchain technologies                   
p   llvm-4.0-dev                                        - Modular compiler and toolchain technologies, libraries and hea
p   llvm-4.0-dev:i386                                   - Modular compiler and toolchain technologies, libraries and hea
p   llvm-4.0-doc                                        - Modular compiler and toolchain technologies, documentation    
p   llvm-4.0-examples                                   - Modular compiler and toolchain technologies, examples         
p   llvm-4.0-runtime                                    - Modular compiler and toolchain technologies, IR interpreter   
p   llvm-4.0-runtime:i386                               - Modular compiler and toolchain technologies, IR interpreter   
p   llvm-4.0-tools                                      - Modular compiler and toolchain technologies, tools            
p   llvm-4.0-tools:i386                                 - Modular compiler and toolchain technologies, tools            
p   llvm-5.0                                            - Modular compiler and toolchain technologies                   
p   llvm-5.0:i386                                       - Modular compiler and toolchain technologies                   
p   llvm-5.0-dev                                        - Modular compiler and toolchain technologies, libraries and hea
p   llvm-5.0-dev:i386                                   - Modular compiler and toolchain technologies, libraries and hea
p   llvm-5.0-doc                                        - Modular compiler and toolchain technologies, documentation    
p   llvm-5.0-examples                                   - Modular compiler and toolchain technologies, examples         
p   llvm-5.0-runtime                                    - Modular compiler and toolchain technologies, IR interpreter   
p   llvm-5.0-runtime:i386                               - Modular compiler and toolchain technologies, IR interpreter   
p   llvm-5.0-tools                                      - Modular compiler and toolchain technologies, tools            
p   llvm-5.0-tools:i386                                 - Modular compiler and toolchain technologies, tools            
i A llvm-6.0                                            - Modular compiler and toolchain technologies                   
p   llvm-6.0:i386                                       - Modular compiler and toolchain technologies                   
i A llvm-6.0-dev                                        - Modular compiler and toolchain technologies, libraries and hea
p   llvm-6.0-dev:i386                                   - Modular compiler and toolchain technologies, libraries and hea
p   llvm-6.0-doc                                        - Modular compiler and toolchain technologies, documentation    
p   llvm-6.0-examples                                   - Modular compiler and toolchain technologies, examples         
i A llvm-6.0-runtime                                    - Modular compiler and toolchain technologies, IR interpreter   
p   llvm-6.0-runtime:i386                               - Modular compiler and toolchain technologies, IR interpreter   
p   llvm-6.0-tools                                      - Modular compiler and toolchain technologies, tools            
p   llvm-6.0-tools:i386                                 - Modular compiler and toolchain technologies, tools            
p   llvm-7                                              - Modular compiler and toolchain technologies                   
p   llvm-7:i386                                         - Modular compiler and toolchain technologies                   
p   llvm-7-dev                                          - Modular compiler and toolchain technologies, libraries and hea
p   llvm-7-dev:i386                                     - Modular compiler and toolchain technologies, libraries and hea
p   llvm-7-doc                                          - Modular compiler and toolchain technologies, documentation    
p   llvm-7-examples                                     - Modular compiler and toolchain technologies, examples         
p   llvm-7-runtime                                      - Modular compiler and toolchain technologies, IR interpreter   
p   llvm-7-runtime:i386                                 - Modular compiler and toolchain technologies, IR interpreter   
p   llvm-7-tools                                        - Modular compiler and toolchain technologies, tools            
p   llvm-7-tools:i386                                   - Modular compiler and toolchain technologies, tools            
i   llvm-dev                                            - Low-Level Virtual Machine (LLVM), libraries and headers       
p   llvm-dev:i386                                       - Low-Level Virtual Machine (LLVM), libraries and headers       
i A llvm-runtime                                        - Low-Level Virtual Machine (LLVM), bytecode interpreter        
p   llvm-runtime:i386                                   - Low-Level Virtual Machine (LLVM), bytecode interpreter 
```

**Comment by pkuczynski** [comment]
Could u please run `rvm get branch /mint-rbx` and try again?

**Comment by bilge-Kagan** [comment]
@pkuczynski 

```
$ rvm --version
rvm 1.29.7-next (mint-rbx) by Michal Papis, Piotr Kuczynski, Wayne E. Seguin [https://rvm.io]
```

```
$ rvm install rbx-3                                                                                            ‹system›
Searching for binary rubies, this might take some time.
No binary rubies available for: mint/19.1/x86_64/rbx-3.107.
Continuing with compilation. Please read 'rvm help mount' to get more information on binary rubies.
Checking requirements for mint.
Installing requirements for mint.
Updating system - please wait
bilgekagan password required for 'apt-get --quiet --yes update': 
Installing required packages: libssl-dev, libssl-dev, clang-3.9, llvm-3.9, llvm-3.9-dev - please wait
Requirements installation successful.
ruby-2.3.1@rubinius - #regenerating binstubs - please wait
rbx-3.107 - #downloading rubinius-3.107, this may take a while depending on your connection...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 15.4M  100 15.4M    0     0  1244k      0  0:00:12  0:00:12 --:--:-- 1515k
rbx-3.107 - #extracting rubinius-3.107 to /usr/share/rvm/src/rbx-3.107 - please wait
rbx-3.107 - #configuring - please wait
Error running 'env /home/bilgekagan/.rvm/wrappers/ruby-2.3.1@rubinius/ruby ./configure --prefix=/usr/share/rvm/rubies/rbx-3 --cc=clang-3.6 --cxx=clang++-3.6 --llvm-config=llvm-config-3.6',
please read /home/bilgekagan/.rvm/log/1552790194_rbx-3.107/configure.log

```

`$ vim /home/bilgekagan/.rvm/log/1552790194_rbx-3.107/configure.log`
```
------------------------------ Deprecation notice ------------------------------


'configure' is deprecated and will be removed in the future.
Use 'build.sh' to configure, build, package, and install Rubinius.


--------------------------------------------------------------------------------
Checking clang-3.6:./configure:1526:in ``': No such file or directory - clang-3.6 (Errno::ENOENT)
	from ./configure:1526:in `check_tool_version'
	from ./configure:708:in `check_tools'
	from ./configure:1975:in `run'
	from ./configure:2218:in `<main>'

```

I also tried build `rbx-3.107` manually, by newer versions of `llvm` and `clang`, but it was not completed; it stick at the end stage..
